openmdao/solvers/nl_gauss_seidel.py

```diff
@@ -137,6 +151,46 @@ def solve(self, params, unknowns, resids, system, metadata=None):
             normval = resids.norm()
             u_norm = np.linalg.norm(unknowns.vec - unknowns_cache)
 
+            if self.options['use_aitken']: # If Aitken acceleration is enabled
```

Comment:
Only thing I would suggest is to add a test that uses this, even if it is something simple like the Sellar MDA. This is just to make sure that nothing in the future breaks this feature.